### DISCUSSION TITLE
Should we integrate docs/guides/blog with visualizer?

### DISCUSSION THREAD

**Discussion by liammulh** [numberscope/frontscope#236] [Ideas]
It might be nice/cool to have a dynamic visualizer in the docs/guide to a visualizer. It might also be nice for future blog posts.

We might want to do something with iframes: https://forum.vuejs.org/t/iframes-in-vue/102268. We would probably have to abandon our static HTML docs, which would be a shame since we spent so much time setting it up.

**Comment by gwhitney** [comment]
At the very least, once it is possible to create links that go directly to a specific active visualization, such links should be (automatically?) generated and inserted into each visualizer doc page, perhaps as the link on the sample image.
